docs: create new otter project (#2469)
AmadeusITGroup/otter: docs/core/START_NEW_APPLICATION.md

  * This is used to run tests and generate distributable files. We strongly encourage to use an up-to-date LTS version of Node.js to ensure the support of all the Otter packages.
    Each package comes with a minimum Node.js version range defined in the `engine` property of its package.json file.

## Creating a new Otter project

```shell
npm create @o3r my-project
# Or a project using yarn package manager
npm create @o3r my-project -- --package-manager=yarn
```

> ### Generate a new application in the project

Reviewer: ```suggestion
### Generate a new application, library or SDK in the project
```

> After creating the project, you can generate a new application, library or sdk within it using the following commands:

Reviewer: ```suggestion
After creating the project, you can generate a new application, library or SDK within it using the following commands:
```

  * This is used to run tests and generate distributable files. We strongly encourage to use an up-to-date LTS version of Node.js to ensure the support of all the Otter packages.
    Each package comes with a minimum Node.js version range defined in the `engine` property of its package.json file.

> ## Creating a new Otter-based project

Reviewer 2: ```suggestion
## Creating a new Otter project
```

```shell
npm create @o3r my-project
# Or a project using yarn package manager

> npm create @o3r my-project -- --package-manager=yarn

Reviewer 3: 👍

```

### Generate a new application in the project

> After creating the project, you can generate a new application within it using the following command:
> 
> ```shell
> ng generate application my-webapp

Reviewer 2: ```suggestion
After creating the project, you can generate a new application/library/sdk within it using the following command:

```shell
# application:
ng generate application my-webapp

# library 
ng generate library my-library

# SDK 
ng generate sdk my-sdk
```

> ng generate application my-webapp

Reviewer 4: This should probably be `yarn ng` or `npm run ng`, otherwise there is a chance that the output will be `ng is not recognized...` or it will use the global `@angular/cli`

Author: yes, I know, I wanted something generic. I think we can stick with `npm` everywhere in our doc. Yarn users will know how to adapt the command :-)
